addons/binding/org.openhab.binding.draytonwiser/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,429 @@
+            <channel id="currentSetPoint" typeId="currentSetPoint-channel"/>
+            <channel id="currentBatteryVoltage" typeId="batteryVoltage-channel"/>
+            <channel id="currentBatteryLevel" typeId="batteryLevel-channel"/>
+            <channel id="currentUnifiedBatteryLevel" typeId="unifiedBatteryLevel-channel"/>
+            <channel id="currentSignalRSSI" typeId="signalRSSI-channel" />
+            <channel id="currentSignalLQI" typeId="signalLQI-channel" />
+            <channel id="currentSignalStrength" typeId="signalStrength-channel" />
+            <channel id="currentUnifiedSignalStrength" typeId="unifiedSignalStrength-channel" />
+            <channel id="zigbeeConnected" typeId="zigbeeConnected-channel" />
+            <channel id="deviceLocked" typeId="deviceLocked-channel" />
+       </channels>
+        
+        <config-description>
+            <parameter name="serialNumber" type="text">
+                <label>Serial Number</label>
+                <description>
+                    Device Serial Number.
+                </description>
+                <required>true</required>
+            </parameter>
+        </config-description>
+    </thing-type>
+    
+    <thing-type id="itrv">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="heathub" />
+        </supported-bridge-type-refs>
+        
+        <label>iTRV</label>
+        <description>Thermostatic Radiator Valve</description>
+       
+        <channels>
+            <channel id="currentTemperature" typeId="currentTemperature-channel"/>
+            <channel id="currentSetPoint" typeId="currentSetPoint-channel"/>
+            <channel id="currentDemand" typeId="demandPercent-channel"/>
+            <channel id="currentBatteryVoltage" typeId="batteryVoltage-channel"/>
+            <channel id="currentBatteryLevel" typeId="batteryLevel-channel"/>
+            <channel id="currentUnifiedBatteryLevel" typeId="unifiedBatteryLevel-channel"/>
+            <channel id="currentSignalRSSI" typeId="signalRSSI-channel" />
+            <channel id="currentSignalLQI" typeId="signalLQI-channel" />
+            <channel id="currentSignalStrength" typeId="signalStrength-channel" />
+            <channel id="currentUnifiedSignalStrength" typeId="unifiedSignalStrength-channel" />
+            <channel id="zigbeeConnected" typeId="zigbeeConnected-channel" />
+            <channel id="deviceLocked" typeId="deviceLocked-channel" />
+        </channels>
+        
+        <config-description>
+            <parameter name="serialNumber" type="text">
+                <label>Serial Number</label>
+                <description>
+                    Device Serial Number.
+                </description>
+                <required>true</required>
+            </parameter>
+        </config-description>
+    </thing-type>
+
+    <!-- Channel Types -->
+    <channel-type id="currentTemperature-channel">
+        <item-type>Number</item-type>
```

Comment:
You can make use of the units of measurement feature here so people can have the value displaye in Fahrenheit for example: Just use `Number:Temperature` here and send `QuantityType`s to the channel. For details, please see: https://www.eclipse.org/smarthome/blog/2018/02/22/units-of-measurement.html